pkg/storage/config/config.go

```diff
@@ -0,0 +1,30 @@
+package config
+
+import (
+	"os"
+)
+
+type Config struct {
+	StoreType     string
+	TokenFilePath string
+}
+
+const (
+	k8s  = "k8s_secrets"
+	none = "none"
+)
+
+// should we check here what the ENV says and act accordingly with values?
+func NewFromEnv() (*Config, error) {
+	storeType := none
+	tokenFilePath := "run/conjur/access-token"
+	if os.Getenv("SECRETS_DESTINATION") == k8s {
```

Comment:
Add "TODO" comment how it should be implemented according to the design: 
k8s - new logic
none or not exists - old logic
incorrect - error

Reply:
please let's not call it "old logic" and "new logic"